app/routes/teams/$id.tsx

```diff
@@ -98,6 +99,45 @@ function WeatherCard({ weather }: { weather: any }) {
   );
 }
 
+function MidpointItem({ team }: { team: Team }) {
+  const fetcher = useFetcher();
+  React.useEffect(() => {
+    if (fetcher.type === "init") {
+      fetcher.load(`/teams/${team.id}/midpoint_weather`);
+    }
+  }, [fetcher]);
+
+  return (
+    <li key={'midpoint-item'}>
```

Comment:
unnecessary interp & mixed quotes here (run prettier?)

Reply:
Yeah I'll see if I can get prettier up and going on nvim